tools/src/links.rs

```diff
@@ -12,26 +12,33 @@ pub fn linkify(s: String) -> String {
         last = link.end();
     }
     if last == 0 {
-        s
+        result = s;
     } else {
         result.push_str(&s[last ..]);
-        result
     }
+
+    linkify_bug_numbers(&result)
```

Comment:
Won't this double-linkify bug numbers?

That is, if I have a commit message like:

```
Foo bar https://bugzilla.mozilla.org/show_bug.cgi?id=1413619
```

Wouldn't this linkify first the bug, and _then_ the bug number again, creating nested links and what not?

Maybe this is good enough though...

Reply:
@emilio The commit message linkification should be unchanged, I just extracted the bug number regex into a separate function for reuse. What you said does apply to `linkify_comment` though; I'll update the regex to also look for the word "bug" which should take care of this.

Reply:
Right, sorry for the confusion, I meant `linkify_comment` ofc. Yeah, probably at least linkify_comment should look for `bug XXX`. For commit messages it may be tricky because people use `b=XXX`, `Bug: XXX` and different patterns.